docs/streams/upgrade-guide.html

```diff
@@ -94,7 +94,14 @@ <h1>Upgrade Guide and API Changes</h1>
 
     <h3><a id="streams_api_changes_300" href="#streams_api_changes_300">Streams API changes in 3.0.0</a></h3>
     <p>
-        A new exception may be thrown from <code>KafkaStreams#store()</code>. If the specified store name does not exist in the topology, an UnknownStateStoreException will be thrown instead of the former InvalidStateStoreException. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-216%3A+IQ+should+throw+different+exceptions+for+different+errors">KIP-216</a> for more information.
+        Interactive Query may throw different new exceptions for different errors:
+    </p>
+    <ul>
+        <li> <code>UnknownStateStoreException</code>: If the specified store name does not exist in the topology, an <code>UnknownStateStoreException</code> will be thrown instead of the former <code>InvalidStateStoreException</code>.</li>
+        <li> <code>StreamsNotStartedException</code>: If Streams state is not <code>REBALANCING</code> or <code>REBALANCING</code>, an <code>StreamsNotStartedException</code> will be thrown instead of the former <code>IllegalStateException</code>.</li>
```

Comment:
```suggestion
        <li> <code>StreamsNotStartedException</code>: If Streams state is not <code>REBALANCING</code> or <code>REBALANCING</code>, a <code>StreamsNotStartedException</code> will be thrown instead of the former <code>IllegalStateException</code>.</li>
```